app/i18n/locale_en.json

```diff
@@ -19,7 +19,9 @@
     "legend.unknown": "Unknown",
     "legend.passrate": "Passrate (%)",
     "legend.ptratio": "Pupil-Teacher Ratio",
-    "filters.title": "Filters",
+    "filters.title": "filters",
```

Comment:
we need to add this new lines in a spreadsheet (shared) so Mark could translate them